src/withdraw-utils.js

```diff
@@ -14,7 +14,7 @@ function hashInputsWithdraw(inputs){
 
     if (
         inputs.rootState === undefined ||
-        inputs.ethAddr === undefined ||
+        inputs.ethAddrState === undefined ||
```

Comment:
There is no need to perform this change.
Furthermore, this change breaks unit test of this function and breaks circuit input naming, so it will break circuit tests of `withdraw.circom`
I will add to pass unit test on `feature/upgrade-v1` besides `master` to avoid this.

Reply:
i'm not agree, i just make a new function `hashInputsWithdrawBjj` that's only valid for this use case!